src/frontend/components/CallAssignment.js

```diff
@@ -142,20 +143,32 @@ class CallAssignment extends React.Component {
   }
 
   renderIntervieweeInfo() {
-    let interviewee = this.props.currentUser.intervieweeForCallAssignment;
-    let name = this.intervieweeName();
-    let number = this.formatPhoneNumber(interviewee.phone)
+    let interviewee = this.props.currentUser.intervieweeForCallAssignment
+    let name = this.intervieweeName()
+    let formattedNumber = this.formatPhoneNumber(interviewee.phone)
+    let plivoStatusText = this.state.plivoStatusText;
+    let plivoCallInProgress = this.state.plivoCallInProgress;
 
     let sideBar = (
-      <div style={{
-        ...BernieText.default,
-        color: BernieColors.blue,
-        fontSize: '1.5em',
-        fontWeight: 600
-      }}>
-        {name}
-        <br />
-        {number}
+      <div>
+        <div style={{
+          ...BernieText.default,
+          color: BernieColors.blue,
+          fontSize: '1.5em',
+          fontWeight: 600
+        }}>
+          {name}
+          <br />
+          <a href={`tel:+1${interviewee.phone}`}
+             style={{color: BernieColors.blue,
+                     textDecoration: 'none'}}>
+            {formattedNumber}
+          </a>
+          <PlivoDialer number={`1${interviewee.phone}`}
+            endpointUsername='bernie151217012337'
```

Comment:
Just adding my comment from the other PR here - I assume these credentials are supposed to just be on the clientside and can't be any more secure than that.  Does plivo protect us by whitelisting our domain?  Also, I was setting up an endpoint on a new account - which application should I be using? Direct Dial?